src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -142,4 +144,22 @@ protected function getInitialSlice(Closure $destination)
 		};
 	}
 
+	/**
+	 * Parse full pipe string to get name and parameters
+	 *
+	 * @param string $pipe
+	 * @return array
+	 */
+	protected  function parsePipeString($pipe)
+	{
+		list($name, $parameters) = array_pad(explode(':', $pipe, 2), 2, []);
+
+		if (is_string($parameters))
+		{
+			$parameters = explode(',', $parameters);
```

Comment:
Shouldn't this use `str_getcsv` (similar to [validation](https://github.com/laravel/framework/blob/5.0/src/Illuminate/Validation/Validator.php#L2109)), so that you can pass quoted parameters with spaces in them?